test/base.test.js

```diff
@@ -268,7 +279,7 @@ test('should create new session if cookie contains invalid session', async (t) =
   const plugin = fastifyPlugin(async (fastify, opts) => {
     fastify.addHook('onRequest', (request, reply, done) => {
       request.sessionStore.set('Qk_XT2K7-clT-x1tVvoY6tIQ83iP72KN', {
-        expires: Date.now() + 1000
+        test: {}
```

Comment:
This test has nothing to do with expiration, so I made the session contain something else. Also, expires lives in the cookie, not top-level